.eslintrc

```diff
@@ -22,7 +22,7 @@
         "eqeqeq": 2,
         "eol-last": 2,
         "indent": [2, 4],
-        "linebreak-style": [2, "unix"],
+        "linebreak-style": [1, "unix"],
```

Comment:
Okay, cool. So what you have seems like the best solution until ESLint does something about it.